bin/i18n/src/i18n/enumerate.clj

```diff
@@ -99,13 +117,16 @@
        (group-by :message)
        (sort-by (comp :file first val))
        (map (fn [[string originals]]
-              {:message string :files (map #(select-keys % [:file :line]) originals)}))))
+              {:message    string
+               :message-pl (->> originals (filter :message-pl) first :message-pl)
+               :files      (map #(select-keys % [:file :line]) originals)}))))
 
 (defn- usage->Message
   "Return a `Message` instance suitable for adding to a `Catalog`"
-  ^Message [{:keys [message files]}]
+  ^Message [{:keys [message message-pl files]}]
```

Comment:
Your PR description mentioned the example where a language has multiple plural forms. But this seems to structurally admit only one plural form. The PR description made me think this was a supported case but I suppose it is just mentioning possible future work then?

Reply:
No, this works currently. You only need to provide one plural form in the macro for English. But when choosing the translated form to return, it will choose from all of the plural options for the target language depending on the value of `n`.

Reply:
(Multiple plural forms for other languages are present in the PO files downloaded from POEditor)

Reply:
Oh I see. So structurally you can only put a single plural in the pot file because English only has one plural form. But in the PO Editor you can enumerate as many as a language might require.